src/broccoli/rollup-with-dependencies.ts

```diff
@@ -15,10 +15,12 @@ class RollupWithDependencies extends Rollup {
   build(...args) {
     let plugins = this.rollupOptions.plugins || [];
     let inputPath = this.inputPaths[0];
+    let packageJSON = JSON.parse(fs.readFileSync(path.join(process.cwd(), 'package.json')).toString('utf8'));
+    let es5 = !('@glimmer/web-component' in packageJSON.devDependencies);
```

Comment:
You don't really need to scan. You can just do `packageJSON.devDependencies['@glimmer/web-component']`.